package.json

```diff
@@ -16,19 +16,32 @@
     "@angular/common": "^11.0.9",
     "@angular/compiler": "^11.0.9",
     "@angular/core": "^11.0.9",
+<<<<<<< HEAD
+=======
+    "@angular/fire": "^6.1.4",
+>>>>>>> auth
     "@angular/forms": "^11.0.9",
     "@angular/material": "^11.0.3",
     "@angular/platform-browser": "^11.0.9",
     "@angular/platform-browser-dynamic": "^11.0.9",
     "@angular/router": "^11.0.9",
+<<<<<<< HEAD
     "ng2-charts": "^2.4.2",
+=======
+    "firebase": "^8.2.2",
+>>>>>>> auth
     "rxjs": "~6.6.0",
     "tslib": "^2.0.0",
     "zone.js": "~0.10.2"
   },
   "devDependencies": {
+<<<<<<< HEAD
     "@angular-devkit/build-angular": "~0.1100.5",
     "@angular/cli": "~11.0.5",
+=======
+    "@angular-devkit/build-angular": "^0.1100.7",
+    "@angular/cli": "^11.0.7",
+>>>>>>> auth
```

Comment:
Do not commit unresolved merge conflicts. Also please do not make updates to the dependencies while in the sprint unless required for functionality. Updates can be their own PRs after the sprint completes.

Reply:
Resolved conflict by removing changed dependency version.